src/main/java/me/zeroeightsix/kami/feature/module/player/Blink.java

```diff
@@ -1,47 +1,79 @@
 package me.zeroeightsix.kami.feature.module.player;
 
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting;
 import me.zero.alpine.listener.EventHandler;
 import me.zero.alpine.listener.Listener;
 import me.zeroeightsix.kami.event.PacketEvent;
 import me.zeroeightsix.kami.feature.module.Module;
+import me.zeroeightsix.kami.util.Texts;
 import net.minecraft.client.network.OtherClientPlayerEntity;
 import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.network.Packet;
 import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket;
+import net.minecraft.util.Formatting;
 
 import java.util.LinkedList;
 import java.util.Queue;
 
+
 @Module.Info(name = "Blink", category = Module.Category.PLAYER)
 public class Blink extends Module {
+    @Setting(name = "PacketDiscard")
```

Comment:
Like 086 said in your other pull, there's no need for a name in the annotation if the name is identical to the original